azext_iot/digitaltwins/_help.py

```diff
@@ -140,6 +140,112 @@ def load_digitaltwins_help():
             az dt reset -n {instance_name}
     """
 
+    helps["dt data-history"] = """
+        type: group
+        short-summary: Manage and configure data history connections.
+    """
+
+    helps["dt data-history create"] = """
+        type: group
+        short-summary: Creates a data history connection between a Digital Twins instance and another resource.
+    """
+
+    helps["dt data-history create adx"] = """
+        type: command
+        short-summary: Creates a data-history connection between a Digital Twins instance and an Azure Database Explorer.
+            Requires pre-created Azure Database Explorer and Eventhub resources.
+        examples:
+        - name: Adds a data history connection to a target instance with the $Default event hub consumer group.
+          text: >
+            az dt data-history create adx -n {instance_name}
+            --cn {time_series_database_connection_name}
+            --adx-cluster-name {adx_cluster_name}
+            --adx-database-name {adx_database_name}
+            --eventhub {event_hub}
+            --eventhub-namespace {event_hub_namespace}
+        - name: Adds a data history connection to a target instance with a custom Azure Database Explorer table name
+            and event hub consumer group.
+          text: >
+            az dt data-history create adx -n {instance_name}
+            --cn {time_series_database_connection_name}
+            --adx-cluster-name {adx_cluster_name}
+            --adx-database-name {adx_database_name}
+            --adx-table-name {adx_table_name}
+            --eventhub {event_hub}
+            --eventhub-namespace {event_hub_namespace}
+            --eventhub-consumer-group {event_hub_consumer_group}
+        - name: Adds a data history connection to a target instance with an Event Hub and Azure
+            Database Explorer instances in different resource groups and subscriptions from the Digital
```

Comment:
Another instance of 'Azure Database Explorer', should be 'Azure Data Explorer' -- also the 'Azure' part is prob not necessary every time.

"Adds a data history connection to a target Digital Twins instance integrating with Event Hub and Data Explorer resources contained in different resource groups or subscriptions to the target Digital Twin**s** instance."?

Reply:
rip me and adx